kubernetes.go

```diff
@@ -8,7 +8,7 @@ import (
 
 	drain "github.com/openshift/kubernetes-drain"
 	corev1 "k8s.io/api/core/v1"
-	"k8s.io/apimachinery/pkg/apis/meta/v1"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
Not really necessary as the dir name _is_ `v1`, but it doesn't hurt.

Reply:
just to be explicit and for clarity since there are 2 `v1`s.